packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -23,8 +23,10 @@ import 'tabs.dart';
 import 'text_theme.dart';
 import 'theme.dart';
 
-const double _kLeadingWidth = kToolbarHeight; // So the leading button is square.
-const double _kMaxTitleTextScaleFactor = 1.34; // TODO(perc): Add link to Material spec when available, https://github.com/flutter/flutter/issues/58769.
+const double _kLeadingWidth =
```

Comment:
save the file without format.

Reply:
@Abhishek01039 could you please clarify more ? what should I do?

Reply:
There are bunch of code formatting changes in this PR - check out the files changed tab.
It seems like your editor ran dartfmt automatically, so I recommend you to turn that off and undo the changes it made. The Flutter framework does not use the standard dartfmt format. :-)
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo